rct229/rulesets/ashrae9012019/ruleset_functions/g311_exceptions/g311_sub_functions/is_space_a_computer_room.py

```diff
@@ -0,0 +1,63 @@
+from pydash import flow
+
+from rct229.rulesets.ashrae9012019.data.schema_enums import schema_enums
+from rct229.rulesets.ashrae9012019.ruleset_functions.baseline_systems.baseline_system_util import (
+    find_exactly_one_space,
+    find_exactly_one_schedule,
+)
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_, assert_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import ZERO, pint_sum
+
+COMPUTER_ROOM_MISC_POWER_DENSITY_THRESHOLD = 20 * ureg("watt/ft2")
+
+LightingSpaceOptions2019ASHRAE901TG37 = schema_enums[
+    "LightingSpaceOptions2019ASHRAE901TG37"
+]
+
+EnergySourceOptions = schema_enums["EnergySourceOptions"]
+
+
+def is_space_a_computer_room(rmi, space_id):
+    space = find_exactly_one_space(rmi, space_id)
+    is_space_a_computer_room_flag = (
+        space.get("lighting_space_type")
+        == LightingSpaceOptions2019ASHRAE901TG37.COMPUTER_ROOM
+    )
+
+    if not is_space_a_computer_room_flag:
+        # define a function that extracts the max value from an hourly_value schedule.
+        get_max_schedule_multiplier_value = flow(
```

Comment:
I love the use of functional programming, but I think this could be more straightforward:
```
function get_max_schedule_multiplier_value(rmi, schedule_id):
    return max(find_exactly_one_schedule(rmi, schedule_id).get("hourly_values", [1.0]))
```

Should this be a utility function we put somewhere else?